src/lib/pubkey/rsa/rsa.cpp

```diff
@@ -413,7 +413,7 @@ class RSA_Private_Operation
          */
          m.sig_words();
 
-         auto future_j1 = std::async(std::launch::async, [this, &m, &d1_mask, powm_window]() {
```

Comment:
This will hit the problem in MSVC where it doesn't realize this capture is not required.

Maybe it is possible to rewrite the code in such a way that Clang doesn't warn but MSVC still builds

Reply:
the easiest fix is to remove const from the variable. Then it becomes a required capture. What do you think?

Reply:
Maybe constexpr would also work for MSVC. I will push and check if it compiles.

Reply:
Re making it non-const kind of ugly. I really want anything that is not modified `const`, since then I don't need to think about if it is modified and if so how... but warning isn't great either.

Based on https://godbolt.org/z/ofmS89 it looks like MSVC accepts if the const is in the global namespace instead of within the function. So I think I'd prefer that. Just rename to `RSA_POWM_WINDOW_BITS` and hoist it out.

Reply:
Looks like constexpr won't work https://godbolt.org/z/q5Irxr

Reply:
...static constexpr works. I think keeping it local like that is preferable.

Reply:
static const/static constexpr

Reply:
static is good